### PR TITLE
Ports radio noises

### PR DESCRIPTION
**Pull request by Coxswain-Navigator**
## About The Pull Request
Ports https://github.com/Quasar-13/Quasar-13/pull/443 on request of our beloved headcoder.

## Why It's Good For The Game
More annoying sounds

## Changelog
:cl:
soundadd: added radio sound effects
/:cl:

### CODE REVIEW

**Review by Kitsunemitsu — DISMISSED** [review]
Port I 100% know works.

**Review by LanceSmites328 — APPROVED** [review]

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
- Make it a preference option to hear those sounds.
- Reduce the volume and range of the sound itself, so that people don't get earblasted from 5 tiles away.

File: code/game/objects/items/devices/radio/radio.dm [hunk 1]
```diff
@@ -221,6 +222,9 @@
 		return
 	if(!M.IsVocal())
 		return
+	if(radio_sounds.len) //Sephora - Radios make small static sounds now.
+		var/sound/radio_sound = pick(radio_sounds)
```
Comment (EgorDinamit) [inline code/game/objects/items/devices/radio/radio.dm:226]: What you are picking isn't a type of sound datum, just a path.
```suggestion
		var/radio_sound = pick(radio_sounds)
```

File: code/game/objects/items/devices/radio/radio.dm [hunk 2]
```diff
@@ -221,6 +222,9 @@
 		return
 	if(!M.IsVocal())
 		return
+	if(radio_sounds.len) //Sephora - Radios make small static sounds now.
```
Comment (EgorDinamit) [inline code/game/objects/items/devices/radio/radio.dm:225]: This is honestly a nitpick, but:
```suggestion
	if(LAZYLEN(radio_sounds)) //Sephora - Radios make small static sounds now.
```
Never use A.len, instead opt in for length(A) proc.

**Comment by Coxswain-Navigator** [comment]
> * 
> * Reduce the volume and range of the sound itself, so that people don't get earblasted from 5 tiles away.

I did the latter, but I believe the former would require me to write a new copy of the radio proc or the playsound() proc, which seems like unnecessary bloat. How would this respect radios that are on the floor with no player otherwise?

**Comment by EgorDinamit** [comment]
What? You just tie preference to sound being sent to the player with it disabled.

**Review by LanceSmites328 — DISMISSED** [review]

**Review by EgorDinamit — APPROVED** [review]
Looks more or less fine; If nothing breaks during test-merge - it is good to go.

**Pull request merged**